### PR TITLE
added: categories

### PR DESCRIPTION
**Pull request by harshalini**
Added categories in Home page.
There are three categories -
1) Men
2) Women
3) Shoes
Screenshot (436)
See the preview here ->
https://6235a8a8b14fd2156af6c1dc--gifted-kare-031385.netlify.app/

### CODE REVIEW

File: src/backend/db/categories.js [hunk 1]
```diff
@@ -11,17 +11,20 @@ export const categories = [
     categoryName: "fiction",
     description:
       "literature in the form of prose, especially novels, that describes imaginary events and people",
+    categoryImg: "./assets/categories/men-category.jpg"
   },
```
Comment (mariyasada) [inline src/backend/db/categories.js]: did you add your category? because i didn't see your category

File: src/components/homeComp.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export { Navbar } from "./shared/navbar";
 export { Header } from "./header/header";
-export { Categories } from "./categories/categories";
-export { Footer } from "./footer/footer";
\ No newline at end of file
+//export { Categories } from "./categories/categories";
```
Comment (mariyasada) [inline src/components/homeComp.js]: remove commented line if it is not useful

**Review by mariyasada — APPROVED** [review]
look at my comments  otherwise code structure is good

File: src/backend/db/categories.js (hunk 1 above)
Comment (harshalini) [reply src/backend/db/categories.js]: Yes will add now, forgot to add

**Review by JantuDeb — APPROVED** [review]
This looks good

**Pull request merged**